src/NzbDrone.Core/Notifications/Kavita/KavitaSettings.cs

```diff
@@ -0,0 +1,46 @@
+﻿using FluentValidation;
+using NzbDrone.Core.Annotations;
+using NzbDrone.Core.ThingiProvider;
+using NzbDrone.Core.Validation;
+
+namespace NzbDrone.Core.Notifications.Kavita;
+
+public class KavitaSettingsValidator : AbstractValidator<KavitaSettings>
+{
+    public KavitaSettingsValidator()
+    {
+        RuleFor(c => c.Host).ValidHost();
+        RuleFor(c => c.Port).InclusiveBetween(1, 65535);
+        RuleFor(c => c.ApiKey).NotEmpty();
+    }
+}
+
+public class KavitaSettings : IProviderConfig
+{
+    private static readonly KavitaSettingsValidator Validator = new KavitaSettingsValidator();
+
+    public KavitaSettings()
+    {
+        Port = 4040;
+    }
+
+    [FieldDefinition(0, Label = "Host")]
+    public string Host { get; set; }
+
+    [FieldDefinition(1, Label = "Port")]
+    public int Port { get; set; }
+
+    [FieldDefinition(3, Label = "API Key", Privacy = PrivacyLevel.ApiKey, HelpLink = "https://wiki.kavitareader.com/en/guides/settings/opds")]
+    public string ApiKey { get; set; }
+
+    [FieldDefinition(4, Label = "Use SSL", Type = FieldType.Checkbox, HelpText = "Connect to Kavita over HTTPS instead of HTTP")]
+    public bool UseSsl { get; set; }
+
+    [FieldDefinition(5, Label = "Have Kavita kick off a scan", Type = FieldType.Checkbox)]
```

Comment:
Can we make `Have Kavita kick off a scan` the HelpText and simplify the label? Something like `Scan` or `Update Library`. This should keep consistency with other library updates (Plex, Kodi, etc)

Reply:
Sounds good, I copied it from Subsonic, but agree `Update Library` is much cleaner

Reply:
Thanks, I'll fix that one to match yours